docs/source/kedro_project_setup/settings.md

```diff
@@ -19,3 +21,21 @@ By default, all code in `settings.py` is commented out. When settings are not su
 | `CONFIG_LOADER_CLASS`       | `kedro.config.ConfigLoader`                       | Customise how project configuration is handled.                                                                    |
 | `CONFIG_LOADER_ARGS`        | `dict()`                                          | Keyword arguments for the `CONFIG_LOADER_CLASS` constructor.                                                       |
 | `DATA_CATALOG_CLASS`        | `kedro.io.DataCatalog`                            | Customise how the [Data Catalog](../data/data_catalog.md) is handled.                                              |
+
+## Settings in `pyproject.toml`
+The `pyproject.toml` file is defined in [PEP 518](https://peps.python.org/pep-0518/) as a configuration file to store build system requirements for Python projects.
+The `pyproject.toml` file in your project contains the metadata for your project and other tool specific configurations.
```

Comment:
Historically yes, but later on it has been extended by PEP 517, and most importantly PEP 621. In addition, many tools started using it to store their settings (Kedro among them, but also most of the ecosystem, including linters etc). Not sure if we should get into specifics here (this is mostly interesting for Python packaging nerds like myself):

```suggestion
The `pyproject.toml` file is the standard way to store build metadata and tool settings for Python projects.
```